cpplite/src/blob/blob_client.cpp

```diff
@@ -184,6 +186,46 @@ std::future<storage_outcome<void>> blob_client::download_blob_to_buffer(const st
     return context->task_promise.get_future();
 }
 
+std::map<std::string, std::string> contentTypeMap {
+	{".css", "text/css"},
+	{".gif",  "image/gif"},
+	{".htm",  "text/html"},
+	{".html", "text/html"},
+	{".jpeg", "image/jpeg"},
+	{".jpg",  "image/jpeg"},
```

Comment:
what about .avi, .wav. .mov, .mp3. , .mp4

Reply:
This list is taken from azcopy, so whatever they support we also added. Other then that they also go on to check first few bytes of file to determine the type but for now we will rely only on extension.